tests/core_tests/startup_tests.py

```diff
@@ -76,17 +76,21 @@ def test(self, timeout: float) -> bool:
 
         return True
 
-
+@pytest.mark.everest_core_config('config-sil.yaml')
```

Comment:
Thats properly the correct way of passing this information into pytest.  Wondering if we should also pass the modules_to_test via these decorators